templates/prelogin/demo.html

```diff
@@ -0,0 +1,126 @@
+{% extends 'prelogin/base.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block stylesheets %}
+    <link href="{% static "app_manager/js/vellum/style.css" %}" type="text/css" rel="stylesheet"/>
+    <style>
+        .form-signup {
+            margin-left: 0;
+        }
+    </style>
+{% endblock stylesheets %}
+
+{% block js-inline %}
+    <script>
+        $(function(){
+            var signUpFormHandler = function() {
+                var submittedForm = $(this);
+                submittedForm.find(".btn[type='submit']").attr('disabled', 'disabled').prepend('<i class="fa fa-refresh fa-spin icon-refresh icon-spin"></i>');
+                kmqPushSafe(['record', 'Clicked "Sign up" from demo', {}, function() {
+                    submittedForm.off('submit', signUpFormHandler).submit();
+                }]);
+                return false;
+            };
+            $('form.form-signup').on('submit', signUpFormHandler);
+        });
+    </script>
+    <script>
+        {% include 'app_manager/partials/form_designer_requirejs_config.html' with vellum_debug=None %}
+
+        require(["jquery", "jquery.vellum"], function ($) {
+            $(function () {
+                $('#formdesigner').vellum({
+                    core: {
+                        loadDelay: 0,
```

Comment:
Pretty sure this should only be set for automated tests. Can't remember what this is used for, but I think it's related to the spinner visibility or something like that. Should be safe to remove this line, causing vellum to use the default value.